server/api/screenshot/screenshotController.js

```diff
@@ -30,13 +30,14 @@ exports.list = function(req, res, next){
 exports.create = function(req, res, next) {
   var username = req.params.username;
   var url = req.body.url;
+  var comment = req.body.comment || "Default comment";
```

Comment:
Remove the default. Comments wouldn't be added on creation of the screenshot; only after. So we shouldn't need to provide that as a parameter.